cmd/corectl/main.go

```diff
@@ -420,6 +423,69 @@ func evictNode(client *rpc.Client, args []string) {
 	dieOnRPCError(err)
 }
 
+func get(client *rpc.Client, args []string) {
+	const usage = "usage: corectl get [key]"
+	if len(args) != 1 {
+		fatalln(usage)
+	}
+
+	req := map[string]interface{}{
+		"keys": []interface{}{args[0]},
+	}
+
+	var resp map[string][][]string
+	err := client.Call(context.Background(), "/config", req, &resp)
+	dieOnRPCError(err)
+	for key, tuples := range resp {
+		if len(tuples) == 1 {
+			fmt.Printf("%s: %s\n", key, strings.Join(tuples[0], " "))
```

Comment:
Since the key is known to the caller, do we have to echo it here? It would be cool if the output were greppable, sortable, etc. What about

```
$ corectl get foo
a1 a2 a3
b1 b3 b3
...
```

with no decoration? Then things like `corectl get foo | sort -k 3` are meaningful and it plays nicer with all the other tools like `uniq` and `join` and `awk`.